### PR TITLE
correct display of Torus provider name

### PR DESCRIPTION
**Pull request by dkent600**
Resolves: https://github.com/daostack/alchemy/issues/1542

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
     "@fortawesome/react-fontawesome": "^0.1.3",
     "@portis/web3": "^2.0.0-beta.43",
     "@sentry/browser": "^5.0.8",
-    "@toruslabs/torus-embed": "^1.2.3",
+    "@toruslabs/torus-embed": "^1.2.6",
```
Comment (orenyodfat) [inline package.json:89]: should this be 1.3.0 ? (latest)

Comment (dkent600) [reply package.json:89]: that is coming in another PR.  No need here.

**Comment by dkent600** [comment]
> duplicate in #1587
> this should be close?

Not a dup

File: package.json (hunk 1 above)
Comment (dkent600) [reply package.json:89]: in any case, 1.30 is badly broken for us.  we have to wait on it.

**Pull request merged**